code/modules/clothing/suits/marine_armor/ert.dm

```diff
@@ -440,6 +440,124 @@
 	armor_internaldamage = CLOTHING_ARMOR_HIGH
 	storage_slots = 2
 
+// UPP Army
+
+/obj/item/clothing/suit/storage/marine/faction/UPP/army
+
+	name = "\improper 6B80 Personal Body Armour"
+	desc = "An older UPP Armour that's since been replaced by the UPP Army Standard 6B90 Body Armour, still used by certain UPP Army units that the UPP doesn't expect to see much combat."
```

Comment:
```suggestion
	desc = "An older UPP personal armor system that's since been replaced by the UPP Army Standard 6B90 body armor. It is still used by certain UPP Army units that the UPP doesn't expect to see much combat."
```